commons/connector/health/aggregation/health_nutrition_measurements.avsc

```diff
@@ -0,0 +1,114 @@
+{
+  "namespace": "org.radarcns.active.healthkit",
+  "name": "HealthKitNutritionMeasurements",
+  "type": ["null", "record"],
+  "default": null,
+  "doc": "User's nutrition data",
+  "fields": [
+    {
+      "name": "time",
+      "type": "double",
+      "doc": "Device timestamp in UTC (s)."
+    },
+    {
+      "name": "timeReceived",
+      "type": "double",
+      "doc": "Time that the data was received from the Apple Healthkit ."
+    },
+    {
+      "name": "nutrition.calories",
+      "type": "double",
```

Comment:
It's not necessarily bad to have these as doubles, but it may be sufficient to have them as `float` instead if you expect each value to have six or fewer digits. Same for all the values below.